dev-python/nbconvert/nbconvert-4.1.0.ebuild

```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
```

Comment:
@marbre don't do implace EAPI bumps in the tree. It's okay in our overlay, but it's against tree policies.